pyincore/analyses/bridgedamage/bridgedamage.py

```diff
@@ -193,6 +210,12 @@ def bridge_damage_analysis_bulk_input(self, bridges, hazard_type,
                         selected_fragility_set.calculate_limit_state(hval_dict,
                                                                      inventory_type="bridge",
                                                                      **bridge_args)
+
+                    if use_liquefaction and geology_dataset_id is not None and liquefaction_resp is not None:
+                        ground_failure_prob = liquefaction_resp[i][BridgeUtil.GROUND_FAILURE_PROB]
```

Comment:
The service returns ground failure expecting 3 LS. The first two limit states use the same probability of ground failure and the last uses the second (you will notice the first two values of ground failure probability are the same). There are actually only 2 values computed for ground failure. One potential workaround would be to do the following after you fetch the liquefaction response on line 215. Add:

# Adjust for 4 LS of bridges - the service returns ground failure expecting 3 LS
ground_failure_prob.append(ground_failure_prob[2])
ground_failure_prob[2] = ground_failure_prob[1]

This will add another ground failure probability to the list and then update the one at index 2 to match what is at 0 and 1. Other suggestions are welcome, but this would be a small change so we don't have to adjust the utility method that handles liquefaction.

Reply:
This is the easiest fix to me. Updated.